.github/workflows/weekly-clean.yml

```diff
@@ -19,7 +19,53 @@ jobs:
       matrix:
         arch: [ARM, ARM_HYP, AARCH64, RISCV64, X64]
         num_domains: ['1', '7', '']
-    steps:
+        include:
```

Comment:
The commit says "tested with the (empty) default for num_domains" but the above says `num_domains: ['1', '7', '']`. I don't understand

Reply:
`include:` is separate from the matrix cross product, if it wanted to include `num_domains`, it would have to list it explicitly e.g.
```
          - arch: ARM_HYP
            plat: exynos5
            num_domains: 7
```
All parameters that are left off are empty, so this will test `num_domains` 1, 7, and 16 (default) for the default platform of ARM, ARM_HYP, AARCH64, RISCV64, X64, and then separately each explicitly listed arch/platform combination, with `num_domains` unset.